server/user-presence.js

```diff
@@ -155,3 +153,6 @@ ServerPresence.onCleanup((serverId) => {
         UserSessions.remove({});
     }
 });
+
+/* eslint-disable import/prefer-default-export */
+export default UserPresence;
```

Comment:
I'm not sure what issue this fixes here. The export seemed to work fine on my end.

Reply:
It was an issue on my side in the past. Will have to check if it still is with the newer versions.